README.md

```diff
@@ -106,7 +106,7 @@ EOF
 Deploy function using Go Knative lambda runtime
 
 ```
-tm deploy service go-lambda -f . --runtime https://raw.githubusercontent.com/triggermesh/knative-lambda-runtime/master/go-1.x/runtime.yaml --wait
+tm deploy service go-lambda -f . --runtime https://raw.githubusercontent.com/triggermesh/knative-lambda-runtime/master/go/runtime.yaml --wait
```

Comment:
should we make this `go1`?

Reply:
I just don't really like how `go1` looks like, plus there are no other go versions out there yet. Idk, If you really feel like it should be `go1` - I comply, otherwise, I'd say "let's wait for Golang 2.0, then Lambda runtime for it, and then rename `go` to `go1`"

Reply:
sounds good. please take a look at https://github.com/triggermesh/knative-lambda-runtime/pull/57